crates/oxc_prettier/src/format/block.rs

```diff
@@ -25,6 +25,20 @@ pub(super) fn print_block<'a>(
     Doc::Array(parts)
 }
 
+pub(super) fn print_statement_block<'a>(
```

Comment:
I'm trying to layout our code to match exactly like prettier. There is no `print_statement_block` in `src/language-js/print/block.js` 🤔

Reply:
> I'm trying to layout our code to match exactly like prettier. There is no `print_statement_block` in `src/language-js/print/block.js` 🤔

Yes, I added this myself.

Reply:
We can hold it until I refactor it